OTKit/otkit-icons/ic_checkbox.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="background: #FFFFFF;">
+    <!-- Generator: Sketch 46.1 (44463) - http://www.bohemiancoding.com/sketch -->
+    <title>icon/ic_checkbox</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
```

Comment:
this example doesn't actually use a def, but some icons we have right now do, and they require a `id` and `xlink`, which seems to conflict sometimes (see https://github.com/opentable/start-page-nodejs/pull/3313/) as an example